week-1/K-functions-parameters/exercise5.js

```diff
@@ -1,5 +1,7 @@
 // Declare your function here
-
+function createLongGreeting(name, number) {
```

Comment:
Just a tiny comment - it's good practice to name variables and parameters what they represent - in this case "number" could be more meaningful named "age" :)